### ISSUE TITLE
[COGNITION]: Names SHOULD not be ALL-CAPS

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8526]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The names of the dependents currently appear in ALL-CAPS, as that is how the information comes from the database. The names **should** not be displayed in ALL-CAPS, as it creates issues for screen readers and people with dyslexia.

If possible, the database should be cleaned.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

- As a dyslexic user, I want to read the names of dependents on my account clearly.
- As a screen reader user, I want to understand the names of dependents on my account clearly.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov/disability/view-dependents/ in browser
2. Log in as user 271
3. Verify the dependent names are ALL-CAPS

## Possible Fixes (optional)

CSS text-transform won't fix this issue, and the information coming from the database should be cleaned.

## WCAG or Vendor Guidance (optional)

1. Some screen readers read capitalized text letter-by-letter. This occasionally even occurs when the HTML is in sentence-case and the CSS forces the capitalization. VoiceOver is especially persnickety about this.
1. "Text in all capital letters is more difficult to read for most people, with and without disabilities." Source: [Accessibility Requirements for People with Low Vision, Section 3.3.4](http://w3c.github.io/low-vision-a11y-tf/requirements.html#capitalization) All capital letters are harder to read (for everyone, but especially people with reading disabilities). The use of All Caps can reduce the readability of your text. Usability expert Jakob Nielsen has found that reading on screen can be around 25% slower than reading from paper, and reading All Caps can be a further 10% slower. It can be used for glance type uses, but it will still be an issue for screen reader users and people with dyslexia.
1. For people with dyslexia, ALL CAPS text may have no difference in shape: all the words are just one big rectangle. Sentence capitals have differentiated shapes based on the letters used. When producing information to be read by others, it is important to remember that up to 10% of your readers may have dyslexia. Dyslexia friendly text will have improved readability and better visual impact for all readers, but especially those with dyslexia.
1. If ALL CAPS must be used, consider using `aria-label="Sentence case words"` so screen readers will read the words rather than spelling out letters. Please validate with all screen readers available to dev. This will still leave issues for those with dyslexia.

## Screenshots or Trace Logs

![Screen Shot 2020-04-28 at 2 07 04 PM](https://user-images.githubusercontent.com/57469/80531691-04018000-8969-11ea-884d-10eb998714c6.png)

**Comment by jason-gcio** [comment]
Checking if this as doable as we think it is

**Comment by tblackwe** [comment]
@zack can you please take a look at where we are presenting names in the 686/674 flow so we can size it more effectively.

**Comment by steele-lm** [comment]
@zack Do we have all the info needed to action this ticket? The specific ask is to address the all-caps list of dependents within the [view dependents](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8526#:~:text=https%3A//staging.va.gov/disability/view%2Ddependents/) page, which is outside the 686 form flow.

**Comment by zack** [comment]
Yes I believe we have the info needed, but if we fix this on the front end we can really only format names as "uppercase first letter, rest lowercase." Which means names like MacDonald will be formatted incorrectly.

**Comment by steele-lm** [comment]
Database change will need to be actioned by BIS. @tblackwe to check with Steve on next steps. Most likely a new feature request for BIS.

**Comment by steele-lm** [comment]
Closing as not actionable by VA.gov bc changing the ALL-CAPS nature of names needs to happen in the system of record.
